packages/pcset/index.d.ts

```diff
@@ -0,0 +1,127 @@
+type Binary = "0" | "1";
+type Chroma = Binary[12];
+type Interval = string;
+/**
+ * Get chroma of a pitch class set. A chroma identifies each set uniquely.
+ * It"s a 12-digit binary each presenting one semitone of the octave.
+ *
+ * Note that this function accepts a chroma as parameter and return it
+ * without modification.
+ *
+ * @param {Array|String} set - the pitch class set
+ * @return {String} a binary representation of the pitch class set
+ * @example
+ * PcSet.chroma(["C", "D", "E"]) // => "1010100000000"
+ */
+export declare function chroma(set: string | string[]): Chroma;
```

Comment:
I'm afraid tonal uses the string representation of the chorma (a 12-length string with 0s and 1s):

```suggestion
export declare function chroma(set: string | Chroma[]): string;
```

As much as I know there's no way to _typeize_ that

Reply:
The type Chroma in this case is an alias to a string type, so this should work. I'm sorry I might be misunderstanding something ?